### PR TITLE
Fix #54

### LINKED ISSUE #54: Remove temporary required symfony/translation
As introduced in #53  the requirement should be removed by using the new version of jms/serializer-bundle

### PR DESCRIPTION
**Pull request by RSalo**
| Q             | A
| ------------- | ---
| Branch?       | "master"
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #54 
| License       | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,14 +12,12 @@
...
         "symfony-cmf/resource-bundle": "^1.0",
-        "jms/serializer-bundle": "^1.0 | ^2.0 | ^3.0",
-        "symfony/translation": "^2.8 || ^3.3 || ^4.0"
+        "jms/serializer-bundle": "^2.0 | ^3.0"
     },
     "require-dev": {
         "symfony-cmf/testing": "^2.1@dev",
         "symfony/phpunit-bridge": "^5",
         "doctrine/phpcr-odm": "^1.4|^2.0",
-        "jms/serializer": "^1.2",
```
Comment (dbu) [inline composer.json]: i think this is the reason why the lowest-version test now fails. which means we should actually put this into the conflict section below, as `"jms/serializer": "<1.2.0"`

Comment (RSalo) [reply composer.json]: But it's already included in the package https://github.com/schmittjoh/JMSSerializerBundle/blob/3.7.0/composer.json#L25

File: composer.json [hunk 2]
```diff
@@ -12,14 +12,12 @@
     "require": {
         "php": "^7.2",
         "symfony-cmf/resource-bundle": "^1.0",
-        "jms/serializer-bundle": "^1.0 | ^2.0 | ^3.0",
-        "symfony/translation": "^2.8 || ^3.3 || ^4.0"
```
Comment (dbu) [inline composer.json:16]: are you sure this is not needed? (totally possible that it was included by mistake, but did you check if any code references things from the Symfony\Components\Translation namespace? i would not trust that the tests in this bundle cover all code)

Comment (dbu) [reply composer.json:16]: sorry, nevermind. just read #54 :man_facepalming:

File: composer.json (hunk 1 above)
Comment (dbu) [reply composer.json]: i agree that we should not have it here as a dev-dependency.

but look at the failing test on travis-ci - they indicate that the visitor interface does not exist. which means that we ended up with a too old version of this package. but should ensure that nobody installs this bundle with an incompatible version, hence we should mark the conflict.

**Comment by dbu** [comment]
oh, this PR includes the commits from #76. shall we wrap things up in #76 and then you rebase this? or we just do both in one MR, i don't actually mind as its no large changes.

**Comment by RSalo** [comment]
@dbu accidentally I did the first PR through the master branch in my fork and another PRs' sticked to this branch :(

File: composer.json (hunk 1 above)
Comment (RSalo) [reply composer.json]: @dbu found out, that only serializer-bundle 3.0 uses jms serializer 2.0. I dropped jms/serializer-bundle 2.0 in a fix. Waiting for travis results...

**Comment by dbu** [comment]
can you please rebase this on master and solve conflicts?

File: composer.json [hunk 3]
```diff
@@ -12,8 +12,7 @@
     "require": {
         "php": "^7.2",
         "symfony-cmf/resource-bundle": "^1.0",
-        "jms/serializer-bundle": "^2.0 || ^3.0",
-        "symfony/translation": "^2.8 || ^3.3 || ^4.0"
+        "jms/serializer-bundle": "^2.0 | ^3.0"
```
Comment (dbu) [inline composer.json]:
```suggestion
        "jms/serializer-bundle": "^2.0 || ^3.0"
```

**Pull request merged**